analyzers/its/expected/ManuallyAddedNoncompliantIssues.CS/S6675-IntentionalFindings-net8.0.json

```diff
@@ -0,0 +1,10 @@
+{
+  "Issues": [
+    {
+      "Id": "S6675",
+      "Message": "Don\u0027t use Trace.WriteLineIf with TraceSwitch level checks.",
```

Comment:
The recommended message in the spec is "Use Trace.WriteXXX instead.". I think we should use that since it instructs the user what to do (instead of what not to do).

Reply:
`"Use Trace.WriteXXX instead."` is not the recommended fix for this issue. [There's a rule](https://github.com/SonarSource/sonar-dotnet/issues/8841) that raises warnings when those methods are called.
[The How to Fix it in for S6675](https://sonarsource.github.io/rspec/#/rspec/S6675/csharp) is more complex, and we can't really decide on which approach to recommend without knowing the context.

Reply:
Hmm, I was looking at the RSPEC and found this:
![image](https://github.com/SonarSource/sonar-dotnet/assets/56015273/53974de2-c312-4fc4-a333-6635135cde59)

If the recommended message is not ok for all cases I think we should update the RSPEC.

Reply:
Thanks. I missed that. Definitely not correct though. I'll update the RSPEC.